src/donejs-carousel-plugin.html

```diff
@@ -9,7 +10,175 @@
 			>
 				{{#each ./slides}}
 
-					<div class="block slide {{#is ../activeSlideIndex scope.index}} active {{/is}}" tabindex="{{#is ../activeSlideIndex scope.index}} 0 {{else}} -1 {{/is}}">
+					<div class="block slide {{#is ../activeSlideIndex scope.index}} active {{/is}}" tabindex="{{#is ../activeSlideIndex scope.index}} 0 {{else}} -1 {{/is}}" style="background-color: {{./backgroundColor}}">
+						
+						<div class="slideContent">
+							<h1>{{./title}}</h1>
+							<p>{{{./content1}}}</p>
+							<p>{{{./content2}}}</p>
+						</div>
+					
+					</div>
+
+				{{/each}}
+
+			</tucows-donejs-carousel>	
+	</script>
+</div>
+
+<div class="demo">
```

Comment:
@ptichonczuk-tc do you know how i could go about not having to duplicate this html code X number of times where X is the number of carousels i would like to have on the demo page?

I'm trying to play around with component fragments and can-stache and can-import but just not having any luck.